ballerina/Ballerina.toml

```diff
@@ -13,7 +13,7 @@ license = ["Apache-2.0"]
 graalvmCompatible = true
 
 [[platform.java17.dependency]]
-groupId = "org.ballerinalang"
+groupId = "io.ballerina.lib"
 artifactId = "redis-utils"
```

Comment:
Can't we rename this to `redis-native` ? I think this is the pattern other modules use.